TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json

```diff
@@ -10,9 +10,9 @@
     ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "NETStandard.Library": "1.5.0-rc3-24113-00"
   },
   "frameworks": {
-    "netstandard1.5": {}
+    "netstandard1.6": {}
```

Comment:
We need to actually version NETStandard.Library to 1.6, I think that is on @ericstj's list. I know there are some new netstandard 1.6 libraries but I would expect there to also be an asset that is less than netstandard1.6 which should have correctly resolved, assuming you aren't depending on the newer APIs. @bartonjs looks like a recent change of your added a netstandard1.4 ref but not a matching implementation. Is that expected?